Detections/SecurityEvent/SecurityEventLogCleared.yaml

```diff
@@ -17,9 +18,10 @@ relevantTechniques:
   - T1107
 query: |
 
-  let timeframe = 1d;
+let timeframe = 1d;
   SecurityEvent
   | where TimeGenerated >= ago(timeframe)
   | where EventID == 1102
+  | where EventSourceName <> "AD FS Auditing" 
```

Comment:
Probably better to do this just in case there is some other ID collision that could occur - 
| where EventID == 1102 and EventSourceName == "Microsoft-Windows-Eventlog"

Reply:
Fixed detection logic. Thank you!